code/__DEFINES/atmospherics/atmos_mapping_helpers.dm

```diff
@@ -14,9 +14,9 @@
 #define BURNMIX_ATMOS "o2=2500;plasma=5000;TEMP=370"
 
 //ATMOSPHERICS DEPARTMENT GAS TANK TURFS
-#define ATMOS_TANK_N2O "n2o=100000;TEMP=293.15" // SKYRAT EDIT - ORIGINAL: "n2o=6000;TEMP=293.15"
-#define ATMOS_TANK_CO2 "co2=100000;TEMP=293.15" // SKYRAT EDIT - ORIGINAL: "co2=50000;TEMP=293.15"
-#define ATMOS_TANK_PLASMA "plasma=100000;TEMP=293.15" // SKYRAT EDIT - ORIGINAL: "plasma=70000;TEMP=293.15"
+#define ATMOS_TANK_N2O "n2o=6000;TEMP=293.15"
+#define ATMOS_TANK_CO2 "co2=50000;TEMP=293.15"
+#define ATMOS_TANK_PLASMA "plasma=70000;TEMP=293.15"
```

Comment:
We're removing these because of gas miners, yeah?

Seeing as how this is the only one of them that's actually going to have an actual in-game impact, I'd recommend you make a changelog entry for this one.